chia/full_node/mempool_manager.py

```diff
@@ -366,8 +362,9 @@ async def add_spend_bundle(
         """
 
         # Skip if already added
-        if spend_name in self.mempool.spends:
-            cost: Optional[uint64] = self.mempool.spends[spend_name].cost
+        existing_item = self.mempool.get_spend_by_id(spend_name)
+        if existing_item is not None:
+            cost: Optional[uint64] = existing_item.cost
             assert cost is not None
             return uint64(cost), MempoolInclusionStatus.SUCCESS, None
```

Comment:
Mempool items don't have an optional cost so this should be simplified by using `existing_item.cost` directly.